v4-client-py/v4_client_py/clients/helpers/chain_helpers.py

```diff
@@ -106,30 +117,30 @@ def calculate_time_in_force(
     time_in_force: OrderTimeInForce, 
     execution: OrderExecution, 
     post_only: bool
```

Comment:
Why is this taken as an argument when it's defined in the `execution` param?

This function is really confusing, ideally we can simplify it since it's not used in the FE and therefore has no dependencies

Reply:
@johnqh ?

Reply:
I don't think i'll touch this to preserve correctness since I'm not entirely familiar with this function. Seems to be related to how we translate FE calls to our chain's proto fields.